readme.md

```diff
@@ -16,74 +26,78 @@ What are the types of collections we have in ruby?
 fruits = ["apple", "banana", "cherry"]
 
 fruits.length # 3
-fruits.push("dates")
+fruits.push("date")
 fruits.length # 4
 
 fruits[0] # apple
-fruits[3] # dates
+fruits[3] # date
 
-fruits[3] = "durian"
-fruits[3] # durian
+fruits[3] = "mango"
+fruits[3] # mango
 
-fruits.join(" ") # "apple banana cherry"
-fruits.join(", ") # "apple, banana, cherry"
-fruits.join(" and ") # "apple and banana and cherry"
+fruits.join(" ") # "apple banana cherry mango"
+fruits.join(", ") # "apple, banana, cherry, mango"
+fruits.join(" and ") # "apple and banana and cherry and mango "
 ```
 
 ### Hashes
 
-Hashes are like javascript objects, but they are a bit more limited:
+Hashes are like Javascript Object Literals, but they are a bit more limited:
 
-```rb
+```ruby
 instructor = {
   name: "Bob",
   age: 30,
-  favorite_foods: ["Cheese Tots", "Cheese Steaks", "Kale Salad"]
+  favorite_foods: ["Tater Tots", "Cheese Steaks", "Kale Salad"]
 }
 
-instructor[:name] # Bob
+instructor[:name] # "Bob"
 instructor[:age]  # 30
-instructor[:favorite_foods] # ["Cheese Tots", "Cheese Steaks", "Kale Salad"]
+instructor[:favorite_foods] # ["Tater Tots", "Cheese Steaks", "Kale Salad"]
 
 instructor[:name] = "Robert"
-instructor[:name] # Robert
+instructor[:name] # "Robert"
 
 instructor[:favorite_color] = "red"
-instructor[:favorite_color] # red
+instructor[:favorite_color] # "red"
 ```
 
```

Comment:
:+1:  on the CFU, was hoping to incorporate the practice in the exercise in the loops section too